source/SIL.AppBuilder.Portal.Frontend/src/lib/fetch.ts

```diff
@@ -2,6 +2,7 @@ import { getToken } from './auth0';
 import { getCurrentOrganizationId } from './current-organization';
 
 export const tryParseJson = async (response) => {
+
```

Comment:
??? also unneeded change

Reply:
It's easier to read if we always add 1 line at the beginning of the function

Reply:
I disagree, if there were a rule for this, I'd add it. :-p
But in general, try not to make changes in your PRs that don't directly affect your PRs.

Reply:
ok will change it, I used to follow some rules from HAPI community